doc/production-solr-collection-swap-process.md

```diff
@@ -1,42 +1,43 @@
 
 # Production Solr Collection Swap Process Steps
 
-This document outlines the expected steps needed to complete the Full Reindex Process to go from exporting data from Alma to having it live in the Production Library Search. This is intended as a checklist for the person responsible for shepherding the process to ensure that the required manual steps are followed.
+This document outlines the expected steps needed to complete the catalog full reindex process from exporting data from Alma to deployment on the production Library Search site. This is intended as a checklist for the person responsible for shepherding the process to ensure that the required manual steps are followed.
 
 These steps are broken down into three sections as these parts can be run sequentially, but they may also be run independently.
 
 ## Full Reindex
 
-1. Initiate Alma FTP Export and wait for completion.
+1. Initiate appropriate Alma SFTP exports via pubishing profiles in Alma and wait for completion.
```

Comment:
Any chance we can document this part further or link to some other documentation that explains what this is?

Reply:
Yes, but I don't think that GitHub is the place to do that. I'll work with Holly to figure out how to best document the Alma parts.

Reply:
If we are leaving this documentation here, then I think it is appropriate to include that.  If someone were using this documentation to go through the process, they need to know there is another step that needs to be done.

Reply:
@dkinzer I'm fine with the alma documentation going somewhere else.  Are you satisfied with that? I will leave it up to you to approve and merge since your question is the one unresolved.

Reply:
Thanks! I promose that I'm working on this Alma documentation and will figure out how best link it with this doc.